tests/layer_tests/pytorch_tests/test_norm.py

```diff
@@ -32,12 +29,87 @@ def forward(self, input_data):
+                super().__init__()
+                self.p = p
+                self.dim = dim
+                self.keepdim = keepdim
+                if self.keepdim is None or self.dim is None:
+                    self.forward = self.forward2
+                else:
+                    self.forward = self.forward4
+
+            def forward4(self, input_data):
+                return input_data.norm(self.p, self.dim, self.keepdim)
+            
+            def forward2(self, input_data):
+                return input_data.norm(self.p)
+
+        ref_net = None
+
+        return aten_norm(p, dim, keepdim), ref_net, "aten::norm"
+
+
     @pytest.mark.nightly
     @pytest.mark.precommit
+    @pytest.mark.parametrize('p', [-2, -1, 0, 1, 2, 2.5, float('inf'), float('-inf')])
+    @pytest.mark.parametrize('dim', [[0], [0, 1], [0, 1, 2]])
+    @pytest.mark.parametrize('keepdim', [True, False])
     def test_norm(self, ie_device, precision, ir_version, p, dim, keepdim):
         self._test(*self.create_model(p, dim, keepdim),
                    ie_device, precision, ir_version)
 
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize('p', [-2, -1, 0, 1, 2, 2.5, float('inf'), float('-inf')])
+    @pytest.mark.parametrize('dim', [None, [0], [0, 1], [0, 1, 2]])
+    @pytest.mark.parametrize('keepdim', [None, True, False])
+    def test_norm_tensor(self, ie_device, precision, ir_version, p, dim, keepdim):
+        self._test(*self.create_model_tensor_norm(p, dim, keepdim),
+                   ie_device, precision, ir_version)
+
+
+class TestFrobeniusNorm(PytorchLayerTest):
+    def _prepare_input(self, out=False):
+        if not out:
+            return (np.random.randn(1, 2, 3).astype(np.float32),)
+        x = np.random.randn(1, 2, 3).astype(np.float32)
+        y = np.zeros_like(x)
+        return (x, y)
+
+    def create_model(self, dim, keepdim, out):
+        class aten_frobenius_norm(torch.nn.Module):
+
+            def __init__(self, dim, keepdim, out) -> None:
+                super().__init__()
+                self.dim = dim
+                self.keepdim = keepdim
+                if out:
+                    self.forward = self.forward_out
+
+            def forward(self, input_data):
+                return torch._VF.frobenius_norm(input_data, self.dim, self.keepdim)
+
+            def forward_out(self, input_data, out):
+                return torch._VF.frobenius_norm(input_data, self.dim, self.keepdim, out=out), out
+
+        ref_net = None
+
+        return aten_frobenius_norm(dim, keepdim, out), ref_net, "aten::frobenius_norm"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize('dim', [(1, ), (0, ), (-1, ), (0, 1), (1, 0)])
+    @pytest.mark.parametrize('keepdim', [True, False])
+    @pytest.mark.parametrize("out", [False, True])
+    def test_frobenius_norm(self, ie_device, precision, ir_version, dim, keepdim, out):
```

Comment:
What if the input to this operation is integer type? Does pytorch support such case?

Reply:
PyTorch does not support such case:
RuntimeError: forbenius_norm(): input dtype should be either floating point or complex. Got Int instead